### ISSUE TITLE
shipit_static_analysis: Include suggested fixes in MozReview comments

### ISSUE THREAD

**Issue by jankeromnes** [mozilla-releng/services#633]
Currently, we don't include an issue's `Body` in MozReview comments, because they mostly duplicate the line of code that the comment is attached to, e.g.:

```
>    } else {

Warning: Do not use 'else' after 'return' [clang-tidy: readability-else-after-return]
    } else {
      ^
```

However, in some cases, the issue `Body` contains a useful fix suggestion, e.g.:

```
>     for (size_t i = 0; i < zStats.notableStrings.length(); i++) {

Warning: Use range-based for loop instead [clang-tidy: modernize-loop-convert]
    for (size_t i = 0; i < zStats.notableStrings.length(); i++) {
    ^
        (const auto & info : zStats.notableStrings)
```

or:

```
>         escapedString.ReplaceSubstring("/", "\\");

Warning: Escaped string literal can be written as a raw string literal [clang-tidy: modernize-raw-string-literal]
        escapedString.ReplaceSubstring("/", "\\");
                                            ^
                                            R"(\)"
```

I think in these cases, it's useful to include the issue `Body`, even if it slightly duplicates the line of code above, because then developers don't need to guess the correct way to address the static analysis issue.

Maybe we could include `Body` in MozReview comments when it contains more than 2 lines?

**Comment by jankeromnes** [comment]
@La0 could you please have a look when convenient for you?

**Comment by jankeromnes** [comment]
Fixed in 57d5e934bb1c9014d4a76642d0730b1a4244d4a4. Thanks!
